content/en/news/releases/1.14.x/announcing-1.14.2/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Announcing Istio 1.14.2
+linktitle: 1.14.2
+subtitle: Patch Release
+description: Istio 1.14.2 patch release.
+publishdate: 2022-07-25
+release: 1.14.2
+---
+
+This release contains bug fixes to improve robustness and some additional support.
+This release note describes what’s different between Istio 1.14.1 and Istio 1.14.2.
+
+FYI, [Go 1.18.4 has been released](https://groups.google.com/g/golang-announce/c/nqrv9fbR0zE),
+which includes 9 security fixes. We recommend you to upgrade to this newer Go version.
```

Comment:
Note we have a comment thread in the 1.13.6 release notes about the Go update as well.

Reply:
Changed to "We recommend you to upgrade to this newer Go version if you are using Go locally."